source/index.txt

```diff
@@ -19,10 +19,17 @@ MongoDB Java Driver
    Release Notes <https://github.com/mongodb/mongo-java-driver/releases>
    View the Source <https://github.com/mongodb/mongo-java-driver>
 
+.. important:: 4.4.0 Beta Documentation
+
+   This is the documentation for the 4.4.0 Beta release of the MongoDB Java Driver,
+   and will become the documentation for the 4.4.0 release. The 4.3 version documentation
+   is available `here <https://docs.mongodb.com/drivers/java/sync/current/>`__.
```

Comment:
I think this is ok as-is, but maybe in the future we can think about using source constants since we might receive requests to update to beta1, beta2, beta3, etc. and there are a couple instances in the docs that we might be able to benefit from the constants.